### PR TITLE
Thread ids

### PR DESCRIPTION
**Pull request by ben-clayton**

### CODE REVIEW

File: gapis/gfxapi/vulkan/vulkan.go [hunk 1]
```diff
@@ -133,8 +132,3 @@ var _ sync.SynchronizedAPI = &api{}
 func (api) GetTerminator(ctx context.Context, c *path.Capture) (transform.Terminator, error) {
 	return NewVulkanTerminator(ctx, c)
 }
-
-// GetDependencyGraphBehaviourProvider implements dependencygraph.DependencyGraphBehaviourProvider interface
-func (api) GetDependencyGraphBehaviourProvider(ctx context.Context) dependencygraph.BehaviourProvider {
-	return newVulkanDependencyGraphBehaviourProvider()
```
Comment (dsrbecky) [inline gapis/gfxapi/vulkan/vulkan.go]: Is this modified by accident as well?

Comment (ben-clayton) [inline gapis/gfxapi/vulkan/vulkan.go]: D'oh! Good catch. Fixed.

**Comment by dsrbecky** [comment]
Rebase please? :-/

**Comment by ben-clayton** [comment]
Rebased

**Pull request merged**